en/docs/administer/key-managers/configure-pingfederate-connector.md

```diff
@@ -0,0 +1,285 @@
+# Configure PingFederate as a Key Manager
+
+WSO2 API Manager supports multiple Key Managers. As a result, WSO2 API Manager is prepacked with an inbuilt resident Key Manager and with the use of connectors it is capable of supporting any authorization server as a Key Manager.
```

Comment:
We have to mention that this OOTB support available only through wum (timestamp)

Reply:
this need to mention the wum timestamp